### DISCUSSION TITLE
GFF size limit

### DISCUSSION THREAD

**Discussion by biouserPD** [GMOD/jbrowse-components#2503] [General] [answered]
Hi,
I tried to open a GFF file in the jbrowse2 desktop version 1.5.1...the file is about 3.6gb and the program showed us the following error:  "Range error:file size (364....)is greater than 2 GB." Is it possible to expand the memory limit of the program?
Or do you suggest other solutions?
Thanks a lot
M

**Comment by cmdcolin** [comment, marked as answer]
Good question. There are some current limits to opening large GFF (there is actually a limit of around ~512MB which we give an error for but apparently there is a separate other issue for bigger than 2GB)

I would suggest making a tabix index for your GFF

To do so, see https://jbrowse.org/jb2/docs/superquickstart_web/#loading-gff3


Basically

```
sudo apt install genometools # or install http://genometools.org/ some other way of interest
gt gff3 -sortlines -tidy -retainids yourfile.gff > yourfile.sorted.gff
bgzip yourfile.sorted.gff
tabix yourfile.sorted.gff.gz
```

You can skip the part where you use `jbrowse add-track` if using the desktop version. This produces

yourfile.sorted.gff.gz and yourfile.sorted.gff.gz.tbi

And you can open this up in the "Add track" workflow

We may end up making a workflow inside jbrowse desktop to perform tabix indexing on the fly so that it is easier to open large files! Let me know if this helps

**Comment by biouserPD** [comment]
Hi,
thanks a lot for your help. I tried your suggestion and I was able to upload the GFF file in Jbrowse.
